vtex/affiliates
# Feature/add install config event

#### What is the purpose of this pull request?

Add a handler that will create a customField "AffiliateId" inside the customData of the orderForm. This is triggered upon installation

#### How to test it?

You can use: `vtex install vtex.affiliates@0.3.1-beta.0`

Wait for the trigger of the event(it can take some minutes) then you can check if it created the customField using:

GET https://{{account}}.vtexcommercestable.com.br/api/checkout/pvt/configuration/orderForm

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/8519076/142211173-fc3cc696-d890-4e93-abb7-ff5f2802fce6.png)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/hugWiqStHQHafTunDr/giphy.gif)

#### Types of changes
